### PR TITLE
Created a Readme file

### PR DESCRIPTION
**Pull request by aclearskye**
Added a read me file to the source code

### CODE REVIEW

**Review by jccit — CHANGES_REQUESTED** [review]
This is a bit of a generic readme, could you make this more about the project?

**Comment by aclearskye** [comment]
Please check again because it looks nicer now :P

File: README [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hey! This file is a readme file for a Wordle clone.
```
Comment (jccit) [inline README]:
```suggestion
This is a simple Wordle clone
```

This is good but I the title should be about the project and not the readme

**Review by jccit — COMMENTED** [review]
This ok, would be nice to see one more change

**Review by jccit — APPROVED** [review]
Ship it 🎉

**Pull request merged**